### PR TITLE
Fix for minimum needs User Interface errors

### LINKED ISSUE #1944: [BUG] Cannot resize Global Minimum Needs Configuration Window
# Problem
I cannot resize Global Minimum Needs Configuration's Window vertically. In the first time the window appear was really big that i need to resize it to find OK button that located in bottom window, but I only can resize the window horizontally and when i want to resize the window vertically, i can't resize it. Here's the screenshot for initial window appear:
inasafe_1

# Platform
I'm running InaSAFE 3.2.0.dev-master in QGIS 2.8.2 with Windows 7 64bit as OS

### PR DESCRIPTION
**Pull request by lucernae**
fix #1944: allows a scrollbar so the user doesn't have to resize window when it exceeds the OS windows

I only add a vertical scrollbar for this commit.

### CODE REVIEW

**Comment by Jannes123** [comment]
This is very strange because on my develop branch the window can resize and  scrollbar is also there.
I'm using ubuntu 14.04 and QGIS2.8

**Comment by lucernae** [comment]
@Jannes123 Wow? is that so? perhaps your develop is ahead of AIFDR develop and haven't been merged yet, so AIFDR develop doesn't have this commit.

In any case, I'm trying to group up other bugs related with minimum needs UI reported by @adissadis

**Comment by Jannes123** [comment]
Please look at PR #1940 deals with 3 bugs

and  PR#1943 which still needs some work

Yes there are changes to my develop which is still work in progress but
those bugfixes were submitted in seperate PRs

**Comment by lucernae** [comment]
That fix in #1940 is great, we should also include it on develop branch, also for PR #1943.

**Comment by lucernae** [comment]
@Jannes123 I will let you handle issue #1933 to merge it to develop since it hasn't been merged

File: safe/gui/tools/minimum_needs/needs_manager_dialog.py [hunk 1]
```diff
@@ -8,6 +8,8 @@
      (at your option) any later version.
 
 """
+from safe.utilities.i18n import tr
```
Comment (akbargumbira) [inline safe/gui/tools/minimum_needs/needs_manager_dialog.py]: Put this import below

**Pull request merged**